lib/l10n_utils/__init__.py

```diff
@@ -106,7 +106,7 @@ def locale_selection(request, available_locales=None):
     return response
 
 
-def render(request, template, context=None, ftl_files=None, activation_files=None, **kwargs):
+def render(request, template, context=None, ftl_files=None, activation_files=None, non_locale_url=False, **kwargs):
```

Comment:
I'm not 100% sure why the 404 and 500 handlers don't need this but we do here. It will be interesting to see how this plays out in production. Do we have a real URL path that we can test this against yet?

Reply:
I was experimenting with reverting [7c8abca](https://github.com/mozilla/bedrock/pull/16469/commits/7c8abcad7ada6551728b31e27302f5f7dde6de8f) so that /help-wanted.html redirects (302) to /{locale}/help-wanted.html which then returns the 410, and that seems to work, returning a localized 410 template and status code. But I'm not sure if that is the behavior that we want, or if I'm missing something else important. Maybe best to wait for @maribedran to get back since her understanding of what's going on here is more complete than mine.